assessments/projects/cni/docs/lightweight-threat-assessment.md

```diff
@@ -0,0 +1,27 @@
+# CNI Lightweight Threat Assessment
```

Comment:
Why is this file included?  (I'm not saying to remove it, but I want to understand why we're adding it as well.)